test/Interop/Cxx/extern-var/Inputs/extern-var.cc

```diff
@@ -0,0 +1,13 @@
+int counter = 0;
+
+int count() {
```

Comment:
WDYT about providing a different API, `getCounterFromCxx()` / `setCounterFromCxx()`? I think it would be more straightforward in tests.

Same for `Namespaced::counter`.

Reply:
Done.